synfig-studio/src/synfigapp/actions/valuedescmakeparenttoactive.cpp

```diff
@@ -0,0 +1,160 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file ValueDescMakeParentToActive.cpp
+**	\brief Template File
+**
+**	$Id$
+**
+**	\legal
+**	......... ... 2013 Ivan Mahonin
```

Comment:
? :)

Reply:
This issue is still present in current code.

Reply:
What should I exactly change here? :sweat_smile:

Reply:
You should write your name instead on "Ivan Mahonin", as you are author of this code. ^__^

Reply:
Ohhh okayy!will do it 😃😃 (I'm exciteddd)

Reply:
Ohhh okayy!will do it 😃😃

Reply:
Looking forward to it! ^__^